roles/README.md

```diff
@@ -7,35 +7,29 @@ XNAT or OMERO.
 
 This collection is tested using Ansible Molecule.
 
-Roles are tested at the collection level rather than as standalone roles, i.e. there is a
-top-level `tests` folder for the collection rather than a `tests/` folder for each role. This
-is to avoid duplicating the Ansible Molecule configuration for each role (especially the
-scenario definitions).
-
-There is a [single converge playbook](../tests/molecule/resources/converge.yml) that by default
-will run all the roles in the playbook. If you would like to run `molecule converge` on a specific
-role instead, you can do this by setting the relevant Ansible tags.
-
-The tags can be set using the
-`MOLECULE_RUN_TAGS` environment variable, which is passed to Ansible Molecule in the
-[scenario definition](../tests/molecule/centos7/molecule.yml) (see the
-`config_options` for the `provisioner` section of the scenario). The accepted tags are defined in the
-[converge playbook](../tests/molecule/resources/converge.yml) for each role.
-
-As an example, to run
-`molecule converge` on the `provision` role only:
-
-```bash
-export MOLECULE_RUN_TAGS=provision
-molecule converge -s centos7
-```
+Each role has its own Molecule configuration, which can be found it the `molecule/`
+folder of each role. Molecule base configurations are used to reduce to amount
+of duplication in the setup for testing each role. There are two base configurations
+that correspond to two [Molecule
+scenarios](https://ansible.readthedocs.io/projects/molecule/getting-started/#molecule-scenarios)
+one for testing on CentOS 7 and another for testing on Rocky 9. The base configurations
+are in the `ansible_collections/mirsg/infrastructure/tests` folder.
+
+Playbooks are tested at the collection level, i.e. their configuration is in the
+top-level `tests` folder for the collection. For each playbook, there are two
+Molecule scenerios - one for CentOS 7 and one for Rocky 9, e.g.
+
+- `ansible_collections/mirsg/infrastructure/tests/molecule/centos7_monitoring`
+- `ansible_collections/mirsg/infrastructure/tests/molecule/rocky9_monitoring`
```

Comment:
Do we need the note about testing playbooks here?

Reply:
oops no, probably not!